hack/windows/Helper.psm1

```diff
@@ -116,8 +116,8 @@ function Install-AntreaAgent {
     }
 
     New-DirectoryIfNotExist $AntreaEtc
-    Get-WebFileIfNotExist $AntreaCNIConfigFile "$AntreaRawUrlBase/build/yamls/windows/base/conf/antrea-cni.conflist"
-    Get-WebFileIfNotExist $AntreaAgentConfigPath "$AntreaRawUrlBase/build/yamls/windows/base/conf/antrea-agent.conf"
+    Get-WebFileIfNotExist $AntreaCNIConfigFile "$AntreaRawUrlBase/main/build/yamls/windows/base/conf/antrea-cni.conflist"
+    Get-WebFileIfNotExist $AntreaAgentConfigPath "$AntreaRawUrlBase/main/build/yamls/windows/base/conf/antrea-agent.conf"
```

Comment:
That doesn't look right. This is how `AntreaRawUrlBase` is set:
```
$AntreaRawUrlBase = "https://raw.githubusercontent.com/$Owner/$Repo/$AntreaVersion"
```
where `AntreaVersion` is a release tag (e.g. `v0.13.0`).
How can this new URL work?